style.css

```diff
@@ -338,3 +366,101 @@ html {
   color: #fff;
   transition: 0.3s;
 }
+
+@media screen and (min-width: 768px) {
+  .container {
+    margin: auto 10%;
+  }
+
+  /* NAVBAR SECTION  */
+  .navbar {
+    padding: 3.2rem 8.8rem;
+  }
+
+  .nav-btn {
+    display: none;
+  }
+
+  .nav-list {
+    list-style: none;
+    display: flex;
+    gap: 3.2rem;
+  }
+
+  .nav-list a:link,
+  .nav-list a:visited {
+    text-decoration: none;
+    color: #344563;
+    font-size: 1.5rem;
+    font-weight: 500;
+  }
+
+  .nav-list a:hover,
+  .nav-list a:active {
+    border-bottom: 0.3rem solid #6070ff;
+  }
+
+  /* HEADLINE SECTION  */
+  .headline {
+    background: url(images/Desktop/Headline-bg.svg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: auto auto;
+    margin: 6.4rem 4.8rem;
+  }
```

Comment:
- [ ] You did a nice job working on the responsiveness of the site. :+1:  Let's work on some more details to match the design. Kindly adjust the background image position as it is not looking like the design for now. The circle on the right side is not fully shown in your design. 🤝️

**Your design**
![Screenshot from 2023-05-04 09-41-15](https://user-images.githubusercontent.com/17243325/236133307-cf210ea7-f508-48f7-8a9f-4b193d450f78.png)

**Figma design**
![Screenshot from 2023-05-04 09-42-38](https://user-images.githubusercontent.com/17243325/236133414-4e52caf9-ccfb-4732-8f6f-8ee3e589c281.png)